package.json

```diff
@@ -120,7 +120,7 @@
     "express-session": "^1.15.6",
     "faker": "^4.1.0",
     "file-saver": "^2.0.2",
-    "form-data": "^2.5.1",
+    "form-data": "^4.0.4",
```

Comment:
Surprised such a bump in versions doesn't cause issues but if it works then that's great